app/controllers/crosswalk_issues_controller.rb

```diff
@@ -2,9 +2,9 @@
 
 class CrosswalkIssuesController < ApplicationController
   def partials
-    @issues = CrosswalkIssue.includes(%i[weam crosswalk ipeds_hd])
+    @issues = CrosswalkIssue.includes(:weam, :crosswalk, :ipeds_hd, weam: :arf_gi_bill)
```

Comment:
i don't think you need to have `:weam` in the list if you have `weam: arf_gi_bill`

Reply:
removed